src/test/java/com/qmetry/qaf/example/steps/StepsLibrary.java

```diff
@@ -1,18 +1,34 @@
 package com.qmetry.qaf.example.steps;
 
-import static com.qmetry.qaf.automation.step.CommonStep.sendKeys;
-
-import org.openqa.selenium.Keys;
+import static com.qmetry.qaf.automation.step.CommonStep.*;
 
 import com.qmetry.qaf.automation.step.QAFTestStep;
 
+import com.qmetry.qaf.automation.ui.webdriver.*;
+
+
 public class StepsLibrary {
+
 	/**
 	 * @param searchTerm
 	 *            : search term to be searched
 	 */
 	@QAFTestStep(description = "search for {0}")
 	public static void searchFor(String searchTerm) {
-		sendKeys(searchTerm+Keys.ENTER, "input.search");
+		sendKeys(searchTerm, "input.search");
+		submit("input.search");
+	}
+
+	/**
+	 * In Europe, a cookies popup appears in google before search is available.
+	 * It has Reject All button. Once it's clicked the search page appears.
+	 * */
+	@QAFTestStep(description = "reject all cookies")
+	public static void rejectAllCookies() {
+		QAFWebElement rejectAllButton = new QAFExtendedWebElement("reject.all");
+
+		if (rejectAllButton.isPresent()) {
+			click("reject.all");
```

Comment:
As you already have element object, better way is to use element object. For instance:
```
rejectAllButton.click();
```